plugins/timesync.koplugin/main.lua

```diff
@@ -1,7 +1,13 @@
 
 local Device = require("device")
 
-if not (Device:isKobo() or Device:isKindle() or Device:isPocketBook()) then
+local command
+-- TODO(hzj-jie): Does pocketbook provide ntpdate?
+if Device:isKobo() then
+    command = "ntpd -q -n -p pool.ntp.org"
+elseif Device:isKindle() or Device:isPocketBook() then
+    command = "ntpdate pool.ntp.org"
```

Comment:
Kindle has a busy box without ntpd. It has its own, more Linux like ntpd, a daemon.